app/templates/courses/courses-view.jade

```diff
@@ -93,6 +93,9 @@ block content
                           if view.showVideosLinkForCourse(course._id)
                             = " "
                             a.view-videos-link(data-classroom-id=classroom.id, data-i18n="courses.view_videos", data-course-id=course._id, data-course-name=i18n(course, 'name'))
+                          // the ranking will only be visible in this class
+                          if courseInstance.id==="5cd4352deed476002dd14019"
```

Comment:
I believe jade template should use `==` instead of `===`.

Reply:
changed on the next commit.